examples/device_twins/main.c

```diff
@@ -140,6 +145,35 @@ static void dt_desired_sample_rate_handler(DX_DEVICE_TWIN_BINDING *deviceTwinBin
     */
 }
 
+// Sample device twin handler that demonstrates how to manage string device twin types.  When an 
```

Comment:
I wanted to add a string copy device twin example here so I can refer to it in my blog.  I see that the other device twins all do something meaningful, report a value or set the sample rate.  I could not think of a good way to fit this into the "theme" of the example, so I just went with an example string handler.  If you have a better Idea, I'm happy to make changes.